shared/general/labels-version-note.md

```diff
@@ -0,0 +1 @@
+__Note:__ If you have devices in your app that have a supervisor version lower than 7.22.0, then you should use the `io.resin.features.` form of the labels to ensure that all devices obey the label. Earlier supervisor versions will not understand the `io.balena.features` label.
```

Comment:
Is it worth adding a link to a page that explains how to determine the supervisor version? :man_shrugging:

Reply:
@lurch do you mean something more than the field that's exposed in the devices dashboard?
![2019-12-08-122203_568x148_scrot](https://user-images.githubusercontent.com/6697261/70393172-79b68080-19b5-11ea-83b2-fd51b74a0932.png)

Reply:
Sorry, I don't actually use balena, so I wasn't aware of what info was displayed in the dashboard! :fearful: 
I'll crawl back into my hole...